### PR TITLE
List titles of random stuff

### LINKED ISSUE #11: Add file for glang's Hometown
- [x] Create a branch
- [x] Make commits
- [x] Create a Pull Request
- [ ] Collaborate
- [ ] Review
- [ ] Merge

### PR DESCRIPTION
**Pull request by glang**
closes #11

### CODE REVIEW

**Comment by pvandyk** [comment]
🥇 💯

File: glang-sf.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# SF
+
+### Places to Eat
+- El farolito
+
+### Places to Visit
+- El farolito
+
+### Places to Avoid
+- jail
```
Comment (pvandyk) [inline glang-sf.md:11]: Excellent option.  👍

File: glang-sf.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# SF
+
+### Places to Eat
+- El farolito
```
Comment (pvandyk) [inline glang-sf.md:4]: Why do you like this place?

File: glang-sf.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+### Places to Visit
+- El farolito
```
Comment (pvandyk) [inline glang-sf.md]: What is special about this place to just visit?

**Review by pvandyk — COMMENTED** [review]
These are general comments.

File: glang-sf.md (hunk 2 above)
Comment (glang) [reply glang-sf.md:4]: Their super burritos and super nachos are soo good. Haven't found another mexican food place that was nearly as good!

**Review by pvandyk — APPROVED** [review]

**Pull request merged**